packages/js/src/core/fetch-token.ts

```diff
@@ -47,7 +47,7 @@ export const fetchTokenByRefreshToken = async (
     body: parameters,
   });
 
-  const scopeValues = tokenSnakeCaseResponse.scope?.split(' ');
+  const finalScopes = tokenSnakeCaseResponse.scope?.split(' ');
```

Comment:
finalScope name stands for？

Reply:
My bad. There is no need to rename it. Reverted.